autoscaling.tf

```diff
@@ -63,7 +63,7 @@ resource "aws_appautoscaling_target" "index_read" {
 
   max_capacity       = each.value["read_max_capacity"]
   min_capacity       = each.value["read_min_capacity"]
-  resource_id        = "table/${aws_dynamodb_table.autoscaled[0].name}/index/${each.key}"
+  resource_id        = "table/${try(aws_dynamodb_table.autoscaled[0].name, aws_dynamodb_table.autoscaled_gsi_ignore[0].name, "")}/index/${each.key}"
```

Comment:
`""` in `try()` is not gonna be used because there is always one table created. Please simplify this bit. :)

Reply:
Sorry about that, fixed :)